diagnostics/WebView2.wprp

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WindowsPerformanceRecorder Version="1.0" Comments="" Company="Microsoft Corporation" Copyright="Microsoft Corporation">
+    <Profiles>
+        <SystemProvider Id="SystemProvider_Light" Base="SystemProvider_Base">
```

Comment:
Try remove `Base="SystemProvider_Base"` and see whether it still works. SystemProvider_Base is not defined anywhere.

Reply:
Done and confirmed it works still.